packages/twenty-server/src/engine/core-modules/user/user.validate.ts

```diff
@@ -0,0 +1,34 @@
+import { User } from 'src/engine/core-modules/user/user.entity';
+import { CustomException } from 'src/utils/custom-exception';
+
+const assertIsExist = (
```

Comment:
assertExists?

Reply:
it means if it does not exist it throws.

Reply:
isExist = return false
assertIsExist = throw